apps/academy/src/components/Header.tsx

```diff
@@ -36,42 +41,44 @@ const PageHeader: FunctionComponent = () => {
   const { pathname } = router;
 
```

Comment:
in fundamentals page the heade behaves weird.. 

![image](https://github.com/user-attachments/assets/45452474-cd17-4d30-97a3-e89fbfbae2be)

the logo at the left looks like with opacity?

Reply:
adjusted the `z-index` of the academy logo to ensure visibility and clickability across all pages